packages/ember-data/lib/serializers/rest2-serializer.js

```diff
@@ -0,0 +1,138 @@
+import RESTSerializer from "ember-data/serializers/rest-serializer";
+import {
+  _normalizeSerializerPayloadItem,
+  pushPayload
+} from "ember-data/system/store/serializer-response";
+
+var forEach = Ember.ArrayPolyfills.forEach;
+//var map = Ember.ArrayPolyfills.map;
+
+function coerceId(id) {
+  return id == null ? null : id + '';
+}
```

Comment:
Looks like this method exists is repeated in the `rest2-serializer` `rest-serializer` and `store` module. Lets move it into its own file and import it in all 3 places.